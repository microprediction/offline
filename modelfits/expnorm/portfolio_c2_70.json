{"g1": 0.0014681603362643111, "g2": 4.173218060515685, "logK": -0.6938760595867371, "loc": 0.002141414372429949, "logScale": 0.6671343692682364}