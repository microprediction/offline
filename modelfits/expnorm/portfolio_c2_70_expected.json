{"g1": 0.21449259673298082, "g2": 12.196626610268934, "logK": 0.4501118451979479, "loc": -0.0934707002651543, "logScale": -1.6314483800701762}